installer/bundle/bundle_skel_AIX.sh

```diff
@@ -0,0 +1,432 @@
+    return $?
+}
+
+# $1 - The filename of the package to be installed
+# $2 - The package name of the package to be installed
+pkg_add() {
+    pkg_filename=$1
+    pkg_name=$2
+
+    echo "----- Installing package: $pkg_name ($pkg_filename) -----"
+    /usr/sbin/installp -a -X -d $pkg_filename ${pkg_name}.rte
+}
+
+# $1 - The package name of the package to be uninstalled
+# $2 - Optional parameter. Only used when forcibly removing omi on SunOS
+pkg_rm() {
+    echo "----- Removing package: $1 -----"
+    /usr/sbin/installp -u $1.rte # 1> /dev/null 2> /dev/null
+}
+
+
+# $1 - The filename of the package to be installed
+# $2 - The package name of the package to be installed
+# $3 - Okay to upgrade the package? (Optional)
+pkg_upd() {
+    pkg_filename=$1
+    pkg_name=$2
+    pkg_allowed=$3
+
+    echo "----- Updating package: $pkg_name ($pkg_filename) -----"
+
+    if [ -z "${forceFlag}" -a -n "$pkg_allowed" ]; then
+        if [ $pkg_allowed -ne 0 ]; then
+            echo "Skipping package since existing version >= version available"
+            return 0
+        fi
+    fi
+
+    [ -n "${forceFlag}" -o "${pkg_name}" = "omi" ] && FORCE="-F" || FORCE=""
+    /usr/sbin/installp -a -X $FORCE -d ${pkg_filename} $pkg_name.rte
+}
+
+#
+# Main script follows
+#
+
+set +e
+
+while [ $# -ne 0 ]
+do
+    case "$1" in
+        --extract-script)
+            # hidden option, not part of usage
+            # echo "  --extract-script FILE  extract the script to FILE."
+            head -${SCRIPT_LEN} "${SCRIPT}" > "$2"
+            local shouldexit=true
+            shift 2
+            ;;
+
+        --extract-binary)
+            # hidden option, not part of usage
+            # echo "  --extract-binary FILE  extract the binary to FILE."
+            tail -n +${SCRIPT_LEN_PLUS_ONE} "${SCRIPT}" > "$2"
+            local shouldexit=true
+            shift 2
+            ;;
+
+        --extract)
+            verifyNoInstallationOption
+            installMode=E
+            shift 1
+            ;;
+
+        --force)
+            forceFlag=true
+            shift 1
+            ;;
+
+        --install)
+            verifyNoInstallationOption
+            installMode=I
+            shift 1
+            ;;
+
+        --purge)
+            verifyNoInstallationOption
+            installMode=P
+            shouldexit=true
+            shift 1
+            ;;
+
+        --remove)
+            verifyNoInstallationOption
+            installMode=R
+            shouldexit=true
+            shift 1
+            ;;
+
+        --restart-deps)
+            restartDependencies=--restart-deps
+            shift 1
+            ;;
+
+        --source-references)
+            source_references
+            cleanup_and_exit 0
+            ;;
+
+        --upgrade)
+            verifyNoInstallationOption
+            installMode=U
+            shift 1
+            ;;
+
+        --version)
+            echo "Version: `getVersionNumber $OM_PKG scx-`"
```

Comment:
Hmm, this was intended as a test helper when I was doing version detection code for Linux. Does this even work properly for UNIX platforms? If it does, leave it. But if it doesn't, just remove it for all UNIX platforms along with the getVersionNumber function. Leave for Linux, of course.

If you prefer, you can fix it to work on all UNIX platforms. But if it doesn't already work, it likely isn't worth it.